README.md

```diff
@@ -51,6 +53,14 @@ This library has a `peerDependencies` listing for [`gatsby`][gatsby].
 
 ## Usage
 
+Use the `options` to configure the script with
+[available attributes](https://developers.pinterest.com/docs/widgets/save/?#button-style-settings).
+
+Note: not all attributes are supported in the plugin yet.
+[See issues for more details](https://github.com/robinmetral/gatsby-plugin-pinterest/issues).
```

Comment:
Thanks for adding this 🙏  We can also revisit this README once all options are supported. If it gets too long we could also extract docs to another page, or even deploy a small example site to Netlify 🙂